include/roaring/roaring.h

```diff
@@ -335,6 +335,13 @@ bool roaring_bitmap_contains_range(const roaring_bitmap_t *r,
                                    uint64_t range_start,
                                    uint64_t range_end);
 
+/**
+ * Check multiple values(vals) sized by n_args.
+ * Returns the results sized by n_args represent the existence of each value.
+ */
+void roaring_bitmap_contains_multi(roaring_bitmap_t *r, size_t n_args,
```

Comment:
I think this should take `const roaring_bitmap_t *`, since it shouldn't modify the bitmap.